<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScopeSelectionWidget</class>
 <widget class="QWidget" name="ScopeSelectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Wybierz zakres zadania:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="scope_combobox"/>
     </item>
     <item>
      <widget class="QPushButton" name="refresh_button">
       <property name="toolTip">
        <string>Odśwież listę</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/options_icons/options_icon_refresh.png</normaloff>:/icons/options_icons/options_icon_refresh.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="disable_scope_checkbox">
     <property name="text">
      <string>Wyłącz ograniczenie zakresem zadania</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="warning_label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string>color: red;</string>
     </property>
     <property name="text">
      <string>UWAGA - FUNKCJA BEZ OGRANICZENIA!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>